examples/world_script_policy.py

```diff
@@ -0,0 +1,139 @@
+# SPDX-FileCopyrightText: ASSUME Developers
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import logging
```

Comment:
why do we need a seperate file for this? We are using config files and input folders for all other examples and adding them into the dict. What is the beenfit of having a seperate file here? Maybe a tutorial notebook would be more beneficial in the future?

Reply:
I like to have some things scripted to not create a lot of csv files additionally.
From my view this makes it easy to see what is going on, when having only a few agents and focusing on the market.
I will create a tutorial for the contracts in the future